public/js/rewarded-ads-manual.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2023 Google LLC
```

Comment:
@niallcalvert Can you please update this to 2024? We're not supposed to do this retroactively for existing files, but new files we are.